src/Utilities/WellKnownTypes.cs

```diff
@@ -290,6 +295,11 @@ public static INamedTypeSymbol String(Compilation compilation)
             return compilation.GetSpecialType(SpecialType.System_String);
         }
 
+        public static INamedTypeSymbol Boolean(Compilation compilation)
```

Comment:
This well known type should never be required - analyzers should directly compare type symbol's SpecialType: `typeSym.SpecialType == SpecialType.System_Boolean`